frontend/src/styles/MarketCard.css

```diff
@@ -105,6 +104,7 @@
 .parallelDateAndHour {
   display: flex;
   align-items: baseline;
+  justify-content: center;
```

Comment:
Ez mit változtat meg? Nem látok különbséget.

Reply:
A stage-n ha rámész a Böba baba-mama vásárra, ott el van csúszva balra az időpont, ezért tettem be.

Reply:
Hát én továbbra sem látom elcsúszva, de ok, ha valami javít, akkor maradjon :)